config/plugin.json

```diff
@@ -13,6 +13,7 @@
 		"shipping-label-banner": true,
 		"store-alerts": true,
 		"wcpay": true,
-		"homescreen": true
+		"homescreen": true,
+		"profiler": true
```

Comment:
I don't think we need to introduce a new feature flag, do we?

Reply:
Good idea! I removed it in the commits 9940b96 and 31c4154.